### ISSUE TITLE
Use Consultation

### ISSUE THREAD

**Issue by SuperCodeCandy** [eslint-stylistic/eslint-stylistic#229]
I'm not sure which template to use to create my question, so I started a new template.

I have a question. When writing the following code, I encountered an error warning under my `id` and `username`, and the editor prompted me with an `Expected indication of 4 spaces but found 2.eslint@stylistic /Js/index`.

```ts
import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';

@Entity()
export class UserEntity {
  @PrimaryGeneratedColumn()
  id: string;

  @Column()
  username: string;
}
```

I'm sorry, but I couldn't find any demonstration or explanation on how to solve this error prompt after reviewing the document. I think this indentation format is an acceptable one. Because, how should I configure it to solve this problem?

**Comment by antfu** [comment]
Duplicated of https://github.com/eslint-stylistic/eslint-stylistic/issues/208
